app/assets/stylesheets/omni_nav/modules/omni-nav.scss

```diff
@@ -176,7 +175,7 @@ html, body { margin:0; padding: 0;}
 
 
         &.expanded {
-
+            z-index: 99998;
```

Comment:
Include a comment here explaining this value. As a rule, try to include comments for magic values like this. Here's a little more on magic numbers:

http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad